### ISSUE TITLE
Feature: Chapter and subtitle backend editor

### ISSUE THREAD

**Issue by benhartwich** [kylegilman/video-embed-thumbnail-generator#338]
Hi,

for now I can upload .vtt files for having chapters, subtitles etc. The available editors for creating such files are mostly very uncomfortable. 
Would it be possible to add a possibility to create fields for subtitles and chapters at the backend directly at the uploaded file?

Regards,

**Comment by kylegilman** [comment]
Yes, you could definitely make something like that. I don't think you'd even need my plugin.

**Comment by benhartwich** [comment]
I think, I´ven´t make it clear: Is it possible at the media file itself at wordpress with your plugin to insert the cue points and texts for chapters and subtitels? There would be no need to use any external vtt editor, which would be great, because it will be very easy for users.

**Comment by kylegilman** [comment]
Oh, are you saying you want me to build this tool for you for free? That could be a premium feature once I get the paid version going, but there are lots of other things I need to do before I can think about building entire new modules from scratch.

**Comment by benhartwich** [comment]
I give you feedback, which could be very helpful for your when you will sell your plugin, because this feature was requested by a video producer. We will buy your plugin, but for now there is no possibility.

**Comment by kylegilman** [comment]
Just because I don't currently require payment for my work doesn't mean I don't appreciate donations, especially considering in addition to the feedback, which I appreciate, you've asked for more individual support than any other user of this plugin by far. I am always open for donations at http://www.kylegilman.net/plugin-donation/ I imagine this University project has a budget larger than zero.

**Comment by benhartwich** [comment]
Yes, you´re right, but german jurisdiction dictates at public institutions, that we only can pay, when we got an invoice to pay for. I´ve already discussed this some time ago with my boss. That´s why I would appreciate your commercial plugin version to give you the appropriate credit.

I think it would be the best, if you can write me further thinkings personally at benjamin.hartwich@uni-passau.de
